index.js

```diff
@@ -43,7 +43,7 @@ function graphIt(){
             options: {
                 title: {
                 display: true,
-                text: 'Statistics since first confirmed national case'
+                text: 'Statistics since first confirmed national case',
```

Comment:
Leaving a comment to test how it works. I think we don't need a comma here since it is the last key-value pair in the object